tests/shell/shell_recorder/README.md

```diff
@@ -1,28 +1,31 @@
 # Shell Recorder
 
+The Shell Recorder is a test tool that checks the output and return values of  shell commands such as `kdb`. It allows you to write unit and  regression tests. The Shell Recorder also records the result of tests in so-called protocol files. These protocol files can be used as basis for new Shell Recorder tests.
+
 ## Configuration
 
-### Mountpoint:
+### Mountpoint
 
-This is the only configuration Variable that has to be set. It is used to determine where the shell_recorder should look for changes.
-e.g. `Mountpoint: user/test` tells the shell_recorder that you will be working under `user/test`.
+This is the only configuration variable that has to be set. It is used to determine where the `shell_recorder` should look for changes.
+e.g. `Mountpoint: user/test` tells the `shell_recorder` that you will be working under `user/test`.
 
-### DiffType:
+### DiffType
 
-Available:
+Use either `Dump` or `Ini` to tell the Shell Recorder to
```

Comment:
Capitalization of plugin names?

Reply:
This was not something I did change. As we discussed in the meeting yesterday, I will remove this feature (nobody uses) anyway.